libs/core/src/lib/card/card.component.ts

```diff
@@ -32,6 +32,8 @@ export class CardComponent implements OnChanges, OnInit, CssClassBuilder, OnDest
     /** Badge */
     @Input() badge: string;
 
+    /** @hidden */
+    _labelText = 'Card with label ';
```

Comment:
This should not be hardcoded. 
1. If the user wants to use other text
2. If the text needs to be translated